### DISCUSSION TITLE
Algunas herramientas para hacer diagramas y planes

### DISCUSSION THREAD

**Discussion by doguedogue** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#37] [Ideas]
- https://app.diagrams.net/
- https://www.bizagi.com/es/plataforma/modeler
- Open Proj https://sourceforge.net/projects/openproj/
- MS Visio https://www.office.com/
- MS Project https://www.office.com/

Espero que les sirvan 🚀

**Comment by LaloCruz** [comment]
También pueden utilizar 

- [LucidChart](https://www.lucidchart.com/pages/usecase/education) con una cuenta de estudiante

**Reply by Eholguin05** [reply]
Yo la hice en Miro, la vdd esta muy accesible la herramienta.
